### PR TITLE
build/configs/rtl8730e : Enable SE configs

### PR DESCRIPTION
**Pull request by jylee9613**
Enable SE configs and NET_SECURITY_TLS

### CODE REVIEW

File: build/configs/rtl8730e/loadable_apps_16M/defconfig [hunk 1]
```diff
@@ -299,7 +299,22 @@ CONFIG_AUTOMOUNT_USERFS=y
...
+CONFIG_SE_AMEBA=y
+CONFIG_HW_RNG=y
+CONFIG_HW_DH_PARAM=y
+CONFIG_HW_ECDH_PARAM=y
+CONFIG_HW_RSA_VERIFICATION=y
+CONFIG_HW_ECDSA_VERIFICATION=y
+CONFIG_HW_RSA_ENC=y
+CONFIG_HW_SE_STORAGE=y
+# CONFIG_HW_AES_ENC is not set
+CONFIG_SE_SECURE_CONTEXT_SIZE=6144
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/loadable_apps_16M/defconfig:317]: Please check and apply the value of the hello.

**Comment by sunghan-chang** [comment]
1. Please check the build break.
2. Please leave why you enable in the description. We dont allow meanless description.

File: build/configs/rtl8730e/loadable_apps_16M/defconfig (hunk 1 above)
Comment (jylee9613) [reply build/configs/rtl8730e/loadable_apps_16M/defconfig:317]: SECURE CONTEXT SIZE is not used in rtl8730e. But I changed the value for consistency.

File: build/configs/rtl8730e/loadable_apps_16M/defconfig [hunk 2]
```diff
@@ -299,7 +302,21 @@ CONFIG_AUTOMOUNT_USERFS=y
...
+CONFIG_SE_AMEBA=y
+CONFIG_HW_RNG=y
+CONFIG_HW_DH_PARAM=y
+CONFIG_HW_ECDH_PARAM=y
+# CONFIG_HW_RSA_VERIFICATION is not set
+CONFIG_HW_ECDSA_VERIFICATION=y
+# CONFIG_HW_RSA_ENC is not set
+CONFIG_HW_SE_STORAGE=y
+# CONFIG_HW_AES_ENC is not set
+CONFIG_SE_SECURE_CONTEXT_SIZE=
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/loadable_apps_16M/defconfig:319]: Is this ok?? There is no value after `=`.

File: build/configs/rtl8730e/loadable_apps_8M/defconfig [hunk 3]
```diff
@@ -1433,6 +1483,8 @@ CONFIG_ENABLE_DATE=y
 # CONFIG_ENABLE_ENV_SET is not set
 # CONFIG_ENABLE_ENV_UNSET is not set
 CONFIG_ENABLE_FREE=y
+CONFIG_ENABLE_HEAPINFO=y
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/loadable_apps_8M/defconfig:1486]: This is not a SE config. Let's enable this with new PR.

File: build/configs/rtl8730e/loadable_apps_8M/defconfig [hunk 4]
```diff
@@ -824,12 +842,24 @@ CONFIG_LWIP_DHCPS_MAX_STATION_NUM=8
...
+CONFIG_TLS_MPI_MAX_SIZE=512
+# CONFIG_TLS_HAVE_NO_TIME_DATE is not set
+
+#
+# HW Options
+#
+CONFIG_TLS_HW_RNG=y
+CONFIG_TLS_HW_DH_PARAM=y
+CONFIG_TLS_HW_ECDH_PARAM=y
+CONFIG_TLS_HW_ECDSA_VERIFICATION=y
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/loadable_apps_8M/defconfig:862]: Hello defconfig does not enable them. Is it ok to enable here?  Could you confirm this?

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**